src/core/Akka.Streams.Tests/Dsl/AsyncEnumerableSpec.cs

```diff
@@ -0,0 +1,120 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="AsyncEnumerableSpec.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2021 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2021 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Akka.Pattern;
+using Akka.Routing;
+using Akka.Streams.Dsl;
+using Akka.Streams.TestKit;
+using Akka.TestKit;
+using FluentAssertions;
+using Nito.AsyncEx.Synchronous;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Akka.Streams.Tests.Dsl
+{
+#if NETCOREAPP
```

Comment:
You can't `await foreach` in the versions of Framework we are targeting with our test runners.